src/Domain/ShoppingList.php

```diff
@@ -24,16 +24,23 @@ class ShoppingList
      */
     private ShoppingItemStack $items;
 
+    /**
+     * @var bool
+     */
+    private bool $isArchived;
+
     /**
      * @param Slug $slug
      * @param string $name
      * @param ShoppingItemStack|null $items
+     * @param bool|false $isArchived
      */
-    public function __construct(Slug $slug, string $name, ShoppingItemStack $items = null)
+    public function __construct(Slug $slug, string $name, bool $isArchived = false, ShoppingItemStack $items = null)
     {
         $this->slug = $slug;
         $this->setName($name);
         $this->items = $items ?? new ShoppingItemStack();
+        $this->isArchived = $isArchived;
```

Comment:
Suggestion: really minor point, we would tend to assign these in the order they are arguments for the constructor. I.e. the `$this->isArchived` assignment should got above the `$this->items` assignment.